### PR TITLE
 [13.0] [MIG] stock_production_lot_firmware_version

### PR DESCRIPTION
**Pull request by Nikul-Chaudhary**
Stock product lot firmware version
==================================
This module allows you to store the current firmware version on an serialized electronic item.

### CODE REVIEW

**Comment by Nikul-Chaudhary** [comment]
@sudhir-erpharbor @aheficent @bodedra can you please review it?

File: stock_production_lot_firmware_version/__manifest__.py [hunk 1]
```diff
@@ -4,16 +4,12 @@
...
     "license": "AGPL-3",
     "author": "Open Source Integrators, Odoo Community Association (OCA)",
     "category": "Stock",
     "website": "https://github.com/OCA/product-attribute",
-    "depends": [
-        "stock"
-    ],
-    "data": [
-        "views/stock_view.xml",
-    ],
```
Comment (bodedra) [inline stock_production_lot_firmware_version/__manifest__.py]: Please avoid cosmetic changes.

File: stock_production_lot_firmware_version/models/stock.py [hunk 2]
```diff
@@ -5,8 +5,6 @@
 
 
 class StockProductionLot(models.Model):
-    _inherit = 'stock.production.lot'
```
Comment (bodedra) [inline stock_production_lot_firmware_version/models/stock.py]: same here.

File: stock_production_lot_firmware_version/models/stock.py [hunk 3]
```diff
@@ -5,8 +5,6 @@
...
+    _inherit = "stock.production.lot"
 
-    firmware_version = fields.Char(
-        string='Firmware Version'
-    )
```
Comment (bodedra) [inline stock_production_lot_firmware_version/models/stock.py]: Same here.

**Review by bodedra — CHANGES_REQUESTED** [review]
@Nikul-Chaudhary Thanks for module migration. Would you please remove your cosmetic changes.

I just do comment just before your code/changes.

File: stock_production_lot_firmware_version/__manifest__.py (hunk 1 above)
Comment (Nikul-Chaudhary) [reply stock_production_lot_firmware_version/__manifest__.py]: @bodedra FYI, Here Pre-commit suggest this kind of changes. That's why I changed code as per    [pre-commit ](https://travis-ci.org/OCA/product-attribute/jobs/601060152)

If not change then Travis is Failed.

Comment (bodedra) [reply stock_production_lot_firmware_version/__manifest__.py]: Ah. Good to know. Thank you.

**Review by bodedra — APPROVED** [review]
LGTM :+1:

**Comment by sudhir-erpharbor** [comment]
Functionally and Code review LGTM.

**Review by sudhir-erpharbor — APPROVED** [review]
Functionally and Code review LGTM.

**Comment by Nikul-Chaudhary** [comment]
@pedrobaeza @max3903 ready to merge

**Pull request merged**